File: examples/features/subscriptions/python/subscribe_url.py
# subscribe_url.py - Subscribe to a service using a URL

from RobotRaconteur.Client import *

url = 'rr+tcp://localhost:29200?service=reynard'
sub = RRN.SubscribeService(url)
c = sub.GetDefaultClientWait(10)
c.say("Hello from URL subscription!")

File: examples/features/subscriptions/cs/subscribe_url.cs
// subscribe_url.cs - Subscribe to a service using a URL

using System;
using RobotRaconteur;
using System.Threading;
using System.Threading.Tasks;
using System.Linq;

// Initialize the client node
using (var node_setup = new ClientNodeSetup(args))
{
    // Subscribe to the service using a URL
    string url = "rr+tcp://localhost:29200?service=reynard";
    var sub = RobotRaconteurNode.s.SubscribeService(url);

    // Get the default client
    var c = (experimental.reynard_the_robot.Reynard)sub.GetDefaultClientWait(10000);

    // Say hello
    c.say("Hello from subscribe_url!");
}
